core/src/test/java/org/apache/servicecomb/core/TestConfig.java

```diff
@@ -53,7 +53,7 @@ public void testConstants() {
     Assert.assertEquals("rest", Const.RESTFUL);
     Assert.assertEquals("", Const.ANY_TRANSPORT);
     Assert.assertEquals("latest", Const.VERSION_RULE_LATEST);
-    Assert.assertEquals("0.0.0+", Const.DEFAULT_VERSION_RULE);
+    Assert.assertEquals("0.0.0.0+", Const.DEFAULT_VERSION_RULE);
```

Comment:
Implement this feature, we must make sure the old version rule works. Can you add new test cases for 4 bit version with change any old test case?

Reply:
As the const DefinitionConst#VERSION_RULE_ALL  is changed from "0.0.0+" to "0.0.0.0+",
I change the value of Const.DEFAULT_VERSION_RULE. you means that I shouldn't change the const 
DefinitionConst#VERSION_RULE_ALL ?

Reply:
I think we do not need to change the VERSION_RULE_ALL constant..